src/config.js

```diff
@@ -15,6 +15,7 @@
 
 let config = {
   APP_ENV: process.env.REACT_APP_ENV,
+  DEPLOYED_ENV: global && global.location ? global.location.hostname.split('.')[0] : 'server',
   ACCOUNTS_URL: process.env.ACCOUNTS_URL || 'https://accounts.openstax.org',
```

Comment:
this seems bad to me.  Do you know of a better alternative?

Reply:
well, webpack will rewrite `global` to be `window` for the web.  Using `global` allows it to also be set for non-dom environments. 

We use the first part of the URL in tutor just so it's shorter but maybe that won't work well with with REX targeted domains.  I'll use the entire hostname for now

Reply:
`typeof(window) === undefined` returns false in node, thats the pattern we follow throughout the code, doesn't rely on using `global`

Reply:
Hm, so using `window` causes specs to fail even though it's behind the typeof check.
```
$ REACT_APP_ENV=test jest --coverage --testPathPattern="(\.|/)spec\.tsx?"
ReferenceError: window is not defined
    at Object.<anonymous> (/Users/nas/code/ox/rex-web/src/config.js:18:59)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
```

We could abandon the attempt at the DEPLOYED_ENV, Sentry will log whatever the url was so we'll still have the info.

Reply:
this is because i led you astray, `typeof` returns a string, so its `typeof(window) === 'undefined'`

Reply:
oh derp, I knew that.  That's what I get for blindly copy & pasting code 😁